gitgud/__main__.py

```diff
@@ -264,12 +267,14 @@ def handle_load(self, args):
 
             if args.level_name is not None:
                 if args.level_name in all_skills[args.skill_name]:
+                    init_tracking_json(self.file_operator)
```

Comment:
You'll be able to remove this once `init_tracking_json` is in `load_level`

Reply:
If i'm not mistaken, you got rid of this whole section in the commit that you made me merge, right?